src/test/java/com/wikia/webdriver/testcases/mercurytests/widgettests/SpotifyTests.java

```diff
@@ -29,51 +29,61 @@ public void prepareTest() {
 
     //@TODO XW-314 - Login is necessary to bypass cache
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)